docs/snippets/common/checkbox-story-csf.js.mdx

```diff
@@ -1,15 +1,15 @@
 ```js
-// Checkbox.stories.js | Checkbox.stories.jsx 
+// Checkbox.stories.js | Checkbox.stories.jsx
```

Comment:
```suggestion
// Checkbox.stories.js | Checkbox.stories.jsx | Checkbox.stories.ts | Checkbox.stories.tsx
```

Reply:
I'm good with this one. Sounds really reasonable, thanks for spotting that!